recipe-client-addon/lib/Storage.jsm

```diff
@@ -21,10 +20,10 @@ function loadStorage() {
   if (storePromise === undefined) {
     const path = OS.Path.join(OS.Constants.Path.profileDir, "shield-recipe-client.json");
     const storage = new JSONFile({path});
-    storePromise = Task.spawn(function* () {
-      yield storage.load();
+    storePromise = (async function () {
```

Comment:
Nah, [JSONFile.load doesn't return anything](https://dxr.mozilla.org/mozilla-central/source/toolkit/modules/JSONFile.jsm#183,211,216) and we want to return the storage object.

On another PR I think I might've refactored this to

```js
storePromise = storage.load().then(() => storage);
```

But I figured for this PR that keeping the translation to async dumb was the way to go.